shopify_integration/invoices.py

```diff
@@ -13,6 +14,10 @@ def prepare_sales_invoice(order, request_id=None):
 
 	try:
 		sales_order = get_shopify_document("Sales Order", cstr(order.get('id')))
+		if not sales_order:
+			sync_sales_order(order, request_id)
```

Comment:
Should this just return the SO itself? Also does it really "sync" or just "create"?

Reply:
They've built the sync function as a webhook trigger, so it doesn't return anything. We can change the name though, yeah.